api/testRunner.yaml

```diff
@@ -0,0 +1,51 @@
+openapi: 3.0.0
```

Comment:
I went with the proposed option @danielbdias suggested of creating a new test runner resource, I think this will allow us to have a more flexible way of dealing with customizable options when executing a test.